### PR TITLE
Migrate deployed sandbox to heroku

### LINKED ISSUE #3125: Sandbox Site is not showing images (404s)
### Issue Summary

When visiting the Sandbox site some of the images are returning a 404

### Steps to Reproduce

1. Visit https://latest.oscarcommerce.com/en-gb/catalogue/
2. Notice most of the images are missing
3. Open up the Javascript Console and you will 404s

The site should load images and not show 404s. ;) 

Screen Shot 2019-08-09 at 11 48 10 AM

### PR DESCRIPTION
**Pull request by solarissmoke**
The sandbox site currently deployed at https://latest.oscarcommerce.com/ is broken, and it seems nobody with the ability to manage/access it is around any more. 

I have set up a new sandbox on Heroku: https://oscar-sandbox.herokuapp.com/, using the configuration that this PR adds. 

Using Heroku for this will also give us the additional advantage of being able to use [review apps](https://devcenter.heroku.com/articles/github-integration-review-apps) for PRs - i.e., to automatically fire up an instance of the sandbox for any PR, to facilitate easy testing.

Because Heroku uses an ephemeral filesystem, any changes made on the deployed sandbox will be cleared periodically without any intervention required.

Fixes #3125.

### CODE REVIEW

File: sandbox/settings.py [hunk 1]
```diff
@@ -119,7 +107,7 @@
 )
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = '$)a7n&o80u!6y5t-+jrd3)3!%vh&shg$wqpjpxc!ar&p#!)n1a'
+SECRET_KEY = env.str('SECRET_KEY', default='$)a7n&o80u!6y5t-+jrd3)3!%vh&shg$wqpjpxc!ar&p#!)n1a')
```
Comment (craigloftus) [inline sandbox/settings.py]: Seems like this line is causing an issue with the sandbox and docs suites, though, on the face of it, I can't see what the issue is.

Comment (solarissmoke) [reply sandbox/settings.py]: Apparently django-environ doesn't handle special characters in strings too well.

Comment (rik) [reply sandbox/settings.py]: A $ prefix is a special case for proxy values: https://django-environ.readthedocs.io/en/latest/#proxy-value

Comment (solarissmoke) [reply sandbox/settings.py]: Good to know, thanks!

**Pull request merged**